unimport/refactor.py

```diff
@@ -41,6 +41,32 @@ def get_location(
     ) -> cst._position.CodeRange:
         return self.get_metadata(cst.metadata.PositionProvider, node)
 
+    def get_rpar(
+        self, rpar: cst.RightParen, location: cst._position.CodeRange
+    ) -> cst.RightParen:
+        return (
```

Comment:
Please omit the node defaults; (example)
```
>>> libcst.TrailingWhitespace()
TrailingWhitespace(
    whitespace=SimpleWhitespace(
        value='',
    ),
    comment=None,
    newline=Newline(
        value=None,
    ),
)
```

Reply:
Yeah, I see that.